src/libraries/IterableMap.sol

```diff
@@ -0,0 +1,104 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+
+contract IterableMap {
+
+    struct Item {
+        bool hasItem;
+        uint256 value;
+        uint256 offset;
+    }
+
+    address owner;
+    bool initialized;
+    uint256[] itemsArray;
+    Item[] itemsMap;
+    uint numberOfItems;
```

Comment:
All variables should have explicit visibility modifiers on them (public, private, internal, external).  For this contract, all of these could be public.